docs/api/algorand/transfer-tokens.mdx

```diff
@@ -4,11 +4,11 @@ Send DeCash from one wallet to another. This endpoint work for both regular and
 
 ## Request Parameters
 
-|     Name    |                Type               | Description |
-| ----------- | --------------------------------- | ----------- |
-| `assetId`   | `string`                          | The ID of the DeCash asset that is being sent. |
-| `sender`    | `{address:string, amount:number}` | Object containing the `address` of the wallet that is sending the funds, and the `amount` that is being sent. Note that this account must have enough funds to cover both the `amount`, and the [fee](#TODO). |
-| `receiver`  | `string`                          | The address of the account that is receiving the funds. Note that, if this account is not a Delegated Wallet, then it must be opted into the token that is being sent. | 
+|     Name    |                   Type                   | Description |
+| ----------- | ---------------------------------------- | ----------- |
+| `assetId`   | `string`                                 | The ID of the DeCash asset that is being sent. |
+| `sender`    | `{address:string, amount:number\|"max"}` | Object containing the `address` of the wallet that is sending the funds, and the `amount` that is being sent. Note that this account must have enough funds to cover both the `amount`, and the [fee](#TODO). If `amount` is set to max, then all of the funds of asset `assetId` are sent to `receiver` (minus the required fee). |
+| `receiver`  | `string`                                 | The address of the account that is receiving the funds. Note that, if this account is not a Delegated Wallet, then it must be opted into the token that is being sent. | 
```

Comment:
Say an account has 0.01 USDD and they request a "max" transfer. What error would the user get?

Reply:
They don't get any error as of right now in that case, an amount of 0 is transferred. If instead the fee was 0.02 and they had a balance of 0.01, then they would get a "not enough funds" error